save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -105,14 +141,30 @@ internal class OrganizationController(
      */
     @PostMapping("/{organizationName}/update")
     @Suppress("UnsafeCallOnNullableType")
-    fun updateOrganization(@RequestBody organization: Organization, authentication: Authentication): Mono<StringResponse> = Mono
-        .just(organization)
-        .filter { organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication, it.name, Role.OWNER) }
-        .map {
-            organizationService.updateOrganization(organization)
+    fun updateOrganization(
+        @RequestBody organization: Organization,
+        authentication: Authentication,
+    ): Mono<StringResponse> = Mono.just(
+        organization.name
+    )
+        .flatMap {
+            organizationService.findByName(organization.name).toMono()
```

Comment:
We are already getting the "organization" object from the frontend. Why do you give a name and look for it again in the database?

Reply:
In case of update we need to be sure that canCreateContests is not changed